behat.yml.dist

```diff
@@ -20,4 +20,4 @@ default:
                 chrome:
                   args:
                     - '--headless'
-                    - '--window-size=1280,720'
+                    - '--window-size=1920,1080'
```

Comment:
I think we should solve this in Behat testing (I think focus is needed), otherwise tests may fail when testing locally with smalled viewport.

Reply:
From what I could see, --window-size switch is used when --headless mode is on only.
Then if window size is set too small in --headless mode, then we got 'button not find 'or 'content not find' errors.
Does not happen when --headless is off.

Reply:
that does not matter, in non-headless mode, we can not expected that the window will be full HD

Reply:
In non-headless, the test pass, no matter what size you set --window-size.
In headless mode, size matters because it seems like window is not scrolling using that mode, which make the test failed. 

If you want to test focusable/scrollable, we should not use headless mode at all.

Reply:
I just tried to resize non-headless Chrome local browser to size like 200x200 and 3 tests failed:
![image](https://user-images.githubusercontent.com/2228672/98271974-d426db00-1f90-11eb-9f33-6eb09219eaff.png)

I will test this more...

Reply:
If I am not using --headless (comment it out) and used window-size of 200,200 then callback feature test pass for me locally. It fail using --headless option.

Pass:
                switches:
                  - "--window-size=200,200"

Fail:
                switches:
                  - "--headless"
                  - "--window-size=200,200"

Reply:
https://github.com/atk4/ui/runs/1359529478 with reverted size, it seems to fail for another reason (empty expected text)

Reply:
Not using window size option make test pass for headless or non-headless.
May be we should remove "--window-size" completely.

Reply:
> https://github.com/atk4/ui/compare/behat_size_debug / https://github.com/atk4/ui/runs/1359529478 with reverted size, it seems to fail for another reason (empty expected text, not bebause something is not clickable)

yes - exactly why I have increase size. Because the container using 'word' is below the window size if reverted.

Reply:
because if may use larger size... Current settings are good I think, as it defines size for screenshotting etc.

Reply:
Current settings is clearly an issue with headless mode. Besides there is no need for screenshots using headless mode. 
I suggest we remove, or increase, window-size value. 
Local setting can be overrides as well.

Reply:
I am strictly againt it. This does not address real issue it only increases size.

The sandpit test above also shown, that very unrelated message is shown.

Reply:
there is definitely an issue in Selenium --headless mode with window size. My guess, only a guess, is when we reload content via callback then something is going with browser window an element text is not accessible after callback.
I have issue a workaround for it. See https://github.com/atk4/ui/pull/1469/commits/59f73d38dc0e905b41aa2e3c1102ecf6cb04a8dd

Reply:
I also wonder why there in no Behat "Stack trace:"...

Reply:
I have include #1538  configuration in this pr for the test to pass temporarily. Lets' resolve that issue over there.